spec/popup/components/header-spec.jsx

```diff
@@ -11,6 +11,7 @@ describe('header', () => {
 
   it('renders a link to the about page', () => {
     const wrapper = shallow(<Header tickets={tickets} />);
+    // eslint-disable-next-line jsx-a11y/anchor-is-valid
```

Comment:
seems strange to me. why do we need this disable?

Reply:
To be honest I did not dig into this, because it was Friday evening. Seems a bit odd to me as well.

Reply:
I will try to take a look later today or tomorrow morning, so we can move on with this.

Reply:
Did you find out why this is needed here?
Tempted to merge this right away, but can't let this be uncommented :P

Reply:
Ah, foo! I missed that one. Here's the answer, will provide a fix ASAP.

Reply:
https://github.com/evcohen/eslint-plugin-jsx-a11y/issues/339

Reply:
Done 👌